test/antrea/types.dl

```diff
@@ -0,0 +1,272 @@
+/*
+ * Types used in internal Antrea NetworkPolicy definition.
+ */
+
+import intern
+import k8spolicy as k8s
+
+// getNormalizedUID generates a unique UUID based on a given string.
+// For example, it can be used to generate keys using normalized selectors
+// unique within the Namespace by adding the constant UID.
+function getNormalizedUID(name: string): string = {
+    var uuid = hash128(name);
+    // TODO: implement proper UUID formatter.
```

Comment:
probably you can just use the substring function to carve pieces from the hex string.

Reply:
We really need DDlog bindings for a proper uuid manipulation lib. Curving pieces will not quite work if the string is wrong length.